Assets/Scripts/Ffb/Dto/ModelChanges/FieldModelAddPushbackSquare.cs

```diff
@@ -4,6 +4,8 @@ public class FieldModelAddPushbackSquare : ModelChange
     {
         public PushbackSquare modelChangeValue;
 
-        public FieldModelAddPushbackSquare() : base("fieldModelAddPushbackSquare") { }
+        public FieldModelAddPushbackSquare() : base("fieldModelAddPushbackSquare")
```

Comment:
This is unnecessary. The constructor calls the base class constructor and doesn't do anything and won't ever do anything beyond that. Leave it as it was.